grails-app/assets/javascripts/exploreArea.js

```diff
@@ -518,39 +517,38 @@ function loadExploreArea(EYA_CONF) {
         // process JSON data
         if(data.length > 0) {
             var lastRow = $('#rightList tbody tr').length;
-            var linkTitle = 'display on map';
             var infoTitle = 'view species page';
             var recsTitle = 'view list of records';
             // iterate over list of species from search
-            for(var i = 0; i < data.length; i++) {
+            var i = 0;
+            data.forEach(function(taxon) {
```

Comment:
``forEach`` callback takes index as a second parameter, no need to maintain it yourself